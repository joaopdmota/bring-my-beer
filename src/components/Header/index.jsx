import React from 'react'
import logo from '../../assets/logo.png'
import Grid from '../Grid'
import styled from 'styled-components'

const Header = styled.header`
  padding: 8px 0;
  ${Grid} {
    display: block;
    text-align: center;
  }
  @media screen and (min-width: 600px) {
    ${Grid} {
      display: flex;
    }
  }
`

export default () => {
  return (
    <Header>
      <Grid justify="space-between" container>
        {/* <img src={logo} width={175} />
        <h1>Bring my beer!</h1> */}
      </Grid>
    </Header>
  )
}
