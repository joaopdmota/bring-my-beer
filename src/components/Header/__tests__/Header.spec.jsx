import React from 'react'
import Header from '../index'

describe('Header component', () => {
  it('render Header default', () => {
    const header = <Header />
    expect(header).toMatchSnapshot()
  })
})
