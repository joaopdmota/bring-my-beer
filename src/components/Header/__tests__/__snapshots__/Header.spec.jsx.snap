// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component render Header default 1`] = `<_default />`;
